node/libs/crypto/src/ed25519/tests.rs

```diff
@@ -0,0 +1,56 @@
+#[test]
+fn test_ed25519() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
anyhow::Result<()>

Reply:
or just () - you use assert (which panic) throughout the test anyway.

Reply:
done

Reply:
The first tests were incorrect. Fixed in https://github.com/matter-labs/era-consensus/pull/86/commits/f600c6d59469b6837f33e456b1d66b38b36b5acd